scripts/broker.csx

```diff
@@ -20,15 +21,26 @@ public void AddUserToVirtualHost() => CreateUserPermissionRequest("PUT", GetBrok
 
 public Broker GetBroker()
 {
+    var connectionString = Environment.GetEnvironmentVariable("RabbitMQTransport.ConnectionString");
+
+    if (string.IsNullOrEmpty(connectionString))
+    {
+        throw new Exception("The `RabbitMQTransport.ConnectionString` environment variable is not set.");
+    }
+
     var connectionStringBuilder = new DbConnectionStringBuilder
     {
-        ConnectionString = Environment.GetEnvironmentVariable("RabbitMQTransport.ConnectionString")
+        ConnectionString = connectionString
     };
 
-    ApplyDefault(connectionStringBuilder, "username", "guest");
-    ApplyDefault(connectionStringBuilder, "password", "guest");
-    ApplyDefault(connectionStringBuilder, "virtualhost", "nsb-rabbitmq-test");
-    ApplyDefault(connectionStringBuilder, "host", "localhost");
+    var missingKeys = new[] { "username", "password", "virtualhost", "host" }
+        .Where(key => !connectionStringBuilder.ContainsKey(key))
+        .ToList();
+
+    if (missingKeys.Any())
```

Comment:
Yep, the reason I did that was that the user has to be explicitly added to the new virtual host, but I guess there's no reason we can't fall back to defaults any way, similar to the tests. I'll change.